src/shared/logger/winstonToolkitLogger.ts

```diff
@@ -46,8 +46,10 @@ export class WinstonToolkitLogger implements Logger, vscode.Disposable {
     }
 
     public setLogLevel(logLevel: LogLevel) {
-        this.logger.info(`Setting log level to: ${logLevel}`)
+        // Log calls are made with explicit levels to ensure the text is output
+        this.logger.log(this.logger.level, `Setting log level to: ${logLevel}`)
         this.logger.level = logLevel
+        this.logger.log(logLevel, `Log level is now: ${this.logger.level}`)
```

Comment:
I feel like this is unnecessary unless there's a reasonable failure case...we should just need the confirmation that it was changed (line 52).

Also, on line 52, use this.logger.level instead of logLevel for confirmation it was changed.

Reply:
Conceptually these are verbose level messages, however, because we can't control the input logLevel, we want to know that this operation is taking place and has taken place.  Using `logLevel` on line 52 is important in verifying this confirmation.